src/lib/components/BodyParser/utils.ts

```diff
@@ -43,7 +43,7 @@ export const regExp = {
   twitter: /^https?:\/\/twitter\.com\/(?:#!\/)?(\w+)\/status(?:es)?\/(\d+)(?:\/.*)?$/,
   symbols: /^[a-zA-Z0-9 ,.";():&%£’€$#@_=/\-'!*?\\/s]{2,}$/,
   image: /.*\.(gif|jpe?g|bmp|png)$/,
-  holdexLink: /^(http:\/\/|https:\/\/)(holdex\.io)/,
+  holdexLink: /^(https?:\/\/(holdex\.io|localhost:3000)|\/[a-zA-Z0-9_-]+(?:\/[a-zA-Z0-9_-]+)*|[a-zA-Z0-9_-]+(?:\/[a-zA-Z0-9_-]+)*)$/,
```

Comment:
1. Why do you need this change? Please explain
2. A more simplified version achieving the same, I believe: 

```suggestion
  holdexLink: /^(https?:\/\/(holdex\.io|localhost:3000)|\/?[\w-]+(?:\/[\w-]+)*)$/,
```

Reply:
1. When I run the project on localhost, the holdeLink RegExp works incorrectly because it doesn’t parse the entire link (e.g., holde.io) or handle localhost:3000.
Here are some example links:
localhost:3000/c/abc/def/xyz
c/companies/truflation
https://holdex.io/companies
https://web.com
localhost:3000/c/abc/xyz
/about/hello
https://holdex.io/about
http://holdex.io/xyz
/about
/xyz/
I originally created the RegExp in a complicated manner.

2. Great solution! It now works for all cases!